.github/workflows/builder.yml

```
@@ -249,7 +249,7 @@ jobs:
     strategy:
       matrix:
         arch: ${{ fromJson(needs.init.outputs.architectures_alpine) }}
-        version: ["3.19", "3.20", "3.21"]
+        version: ["3.19", "3.20", "3.21", "3.22"]
```

```suggestion
        version: ["3.20", "3.21", "3.22"]
```
Only the latest 3 Alpine versions are supported, so `3.19` should be dropped.
Please update the readme section for it as well.

https://github.com/home-assistant/docker-base/blob/1198c43c1a06a90916bb0b0aa118692076e3d460/README.md?plain=1#L26-L28

---

Updated